roles/devscripts/tasks/131_packages.yml

```diff
@@ -28,3 +28,9 @@
   ansible.builtin.import_role:
     name: ci_setup
     tasks_from: epel.yml
+
+- name: Ensure conflicting package does not exist.
+  become: true
+  ansible.builtin.package:
+    name: "ansible-core-2.14.9"
```

Comment:
Good question. It is driven by a foreman template hence the specific one. It would be a problem when the platform moves to the next minor version. Let me go ahead and decouple the version.

Thank you.